tests/avif/Makemodule.am

```diff
@@ -0,0 +1,6 @@
+libgd_test_programs += \
```

Comment:
same here about adding build-time protection when avif isn't enabled

Reply:
thanks for noticing this